src/main/java/org/nikita111100/onlinetesting/controller/PossibleAnswerController.java

```diff
@@ -0,0 +1,64 @@
+package org.nikita111100.onlinetesting.controller;
+
+import org.nikita111100.onlinetesting.model.persistent.PossibleAnswer;
+import org.nikita111100.onlinetesting.model.persistent.Question;
+import org.nikita111100.onlinetesting.service.PossibleAnswerService;
+import org.nikita111100.onlinetesting.service.QuestionService;
+import org.nikita111100.onlinetesting.service.TestService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/possibleAnswers")
+public class PossibleAnswerController {
+    private final PossibleAnswerService possibleAnswerService;
+    private final QuestionService questionService;
+
+    public PossibleAnswerController(PossibleAnswerService possibleAnswerService, QuestionService questionService, TestService testService) {
+        this.possibleAnswerService = possibleAnswerService;
+        this.questionService = questionService;
+    }
+
+    @GetMapping
+    public String findAll(Model model){
+        List<PossibleAnswer> possibleAnswers = possibleAnswerService.findAll();
+        List<Question> questions = questionService.findAll();
+        model.addAttribute("questions", questions);
+        model.addAttribute("possibleAnswers",possibleAnswers);
+        return "possibleAnswer-list";
```

Comment:
```suggestion
        return "possibleAnswers/list";
```
давай и остальные шаблоны сгруппируем по этому правилу. Так шаблоны будут разбиты по привязке к сущности. Шаблоны надо будет перенести в соответствующий каталог.